integration-tests/ccip-tests/testsetups/ccip.go

```diff
@@ -1414,6 +1426,24 @@ func (o *CCIPTestSetUpOutputs) CreateEnvironment(
 	return chainByChainID
 }
 
+func flushClLogs(l zerolog.Logger, testEnv *test_env.CLClusterTestEnv) {
```

Comment:
@Tofel I need your help with that one, it was added in the CCIP fork, but doesn't compile here